src/components/Login/EmailVerification/VerificationResult.tsx

```diff
@@ -1,10 +1,10 @@
 import { ModalHeader, ModalBody, Text, Icon, Button, ModalFooter } from "@chakra-ui/react";
-import { RiArrowRightLine } from "react-icons/ri";
 import { BsCheckCircleFill, BsXCircleFill } from "react-icons/bs";
 import ModalOperator from "../ModalOperater";
+import React from "react";
```

Comment:
I don't think we need to import react anywhere in Nextjs.

Reply:
OK! My IDE always warn me of too many unnecessary things:)